.github/workflows/ci.yml

```diff
@@ -44,3 +44,29 @@ jobs:
 
       - name: Test coverage
         run: npx truffle run coverage
+
+      - name: Upload coverage
+        run: bash <(curl -s https://codecov.io/bash) || true
+
+  static-analysis:
+    name: Static Analysis
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: Install python
```

Comment:
Python is not installed by default on the ubuntu image?

Reply:
Python is, but not `pip3` AFAIK.